<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exop</class>
 <widget class="QMainWindow" name="exop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>exop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_encont_valores">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Encontrar valores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_guardar_arch">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar en archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cargar_db">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar la base de datos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_min1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_min1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Primer mínimo</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_t_brillo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_min2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_min2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Segundo mínimo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tmin2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tmin1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_max">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_max">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_periodo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Periodo orbital [Dias]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_periodo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_radp">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_radio">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio del planeta [Jupiters]</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="grafica_datos" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>461</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_tmin1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo del primer mínimo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tmin2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo del segundo mínimo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
